File: microservices/review/api/src/main/java/fr/edjaz/api/core/review/Review.java
package fr.edjaz.api.core.review;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Review {
    private int productId;
    private int reviewId;
    private String author;
    private String subject;
    private String content;
    private String serviceAddress;


}

File: microservices/review/api/src/main/kotlin/fr/edjaz/api/core/review/Review.kt
package fr.edjaz.api.core.review

data class Review(
  var productId: Int = 0,
  var reviewId: Int = 0,
  var author: String? = null,
  var subject: String? = null,
  var content: String? = null,
  var serviceAddress: String? = null
)
